src/pages/TCY/components/TCYCta.tsx

```diff
@@ -10,6 +10,26 @@ import { AssetIcon } from '@/components/AssetIcon'
 
 const margin = { base: 4, md: 0 }
 
+const CtaSkeleton = () => {
+  return (
+    <Alert status='info' m={margin} width='auto'>
+      <SkeletonCircle size='32px' />
+      <Stack spacing={0} ml={4}>
+        <Skeleton width='100px'>
+          <AlertTitle>Loading...</AlertTitle>
+        </Skeleton>
+      </Stack>
+      <Skeleton ml='auto' flexShrink={0} borderRadius='xl'>
+        <Button ml='auto' flexShrink={0}>
+          Check Claims
```

Comment:
Translation string

Reply:
these are skeletons you never see the text, its there to get the correct size

Reply:
The correct size is dynamic based on the language set though, right?